api/models/templates/app.tmpl

```diff
@@ -382,18 +382,16 @@
       "Condition": "Enabled{{ upper .ProcessName }}",
       "DependsOn": [ "{{ .ResourceName }}SecurityGroup" ],
       "Properties": {
-        "Scheme": { "Fn::If": [ "Internal",
-          "internal",
-          {{ if eq .Scheme "internal" }}
-            "internal"
-          {{ else }}
-            { "Ref": "AWS::NoValue" }
-          {{ end }}
-        ] },
-        "Subnets": { "Fn::If": [ "Private",
-          { "Ref": "SubnetsPrivate" },
-          { "Ref": "Subnets" }
-        ] },
+        {{ if eq .Scheme "internal" }}
+          "Scheme": "internal",
+          "Subnets": { "Fn::If": [ "Private",
+            { "Ref": "SubnetsPrivate" },
+            { "Ref": "Subnets" }
+          ] },
+        {{ else }}
+          "Scheme": { "Fn::If": [ "Internal", "internal", { "Ref": "AWS::NoValue" } ] },
```

Comment:
There is the CF parameter "Internal" and the local check of .Scheme == "internal" (the latter is set when you use only a single port pair)